### PR TITLE
Finish populating schema

### PR DESCRIPTION
**Pull request by Can-Tina**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -12,9 +12,93 @@ datasource db {
...
+  name           String
+  art            String
+  type1          String
+  type2          String?
+  ratingAvg      Int              @default(1)
+  ratingCount    Int              @default(0)
+  createdAt      DateTime         @default(now())
+  updatedAt      DateTime         @updatedAt
+  userTeamMember UserTeamMember[]
+  rating         Rating[]
```
Comment (nathanSamosa) [inline prisma/schema.prisma:25]: ratingCount works, but perhaps there is a way to find the count through the number of rating models related to a Pokemon? Achieves the same thing but with one less key.

File: prisma/schema.prisma [hunk 2]
```diff
@@ -12,9 +12,93 @@ datasource db {
...
+  id        Int      @id @default(autoincrement())
+  post      Post     @relation(fields: [postId], references: [id])
+  postId    Int      @unique
+  profile   Profile  @relation(fields: [profileId], references: [id])
+  profileId Int      @unique
+  parentId  Int?
+  content   String
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+}
```
Comment (TheunisK) [inline prisma/schema.prisma:104]: This all looks really well done Lenni. Struggling to fault it!

File: prisma/schema.prisma [hunk 3]
```diff
@@ -12,9 +12,93 @@ datasource db {
...
+model Profile {
+  id             Int              @id @default(autoincrement())
+  user           User             @relation(fields: [userId], references: [id])
+  userId         Int              @unique
+  createdAt      DateTime         @default(now())
+  updatedAt      DateTime         @updatedAt
+  userTeamMember UserTeamMember[]
+  rating         Rating[]
+  post           Post[]
+  Comment        Comment[]
```
Comment (nathanSamosa) [inline prisma/schema.prisma]: comment is capitalised

File: prisma/schema.prisma [hunk 4]
```diff
@@ -12,9 +12,93 @@ datasource db {
...
+  id         Int       @id @default(autoincrement())
+  profile    Profile   @relation(fields: [profileId], references: [id])
+  profileId  Int       @unique
+  category   Category  @relation(fields: [categoryId], references: [id])
+  categoryId Int       @unique
+  title      String
+  content    String
+  createdAt  DateTime  @default(now())
+  updatedAt  DateTime  @updatedAt
+  Comment    Comment[]
```
Comment (nathanSamosa) [inline prisma/schema.prisma]: comment is capitalised

File: prisma/schema.prisma (hunk 2 above)
Comment (Can-Tina) [reply prisma/schema.prisma:104]: :)

File: prisma/schema.prisma (hunk 1 above)
Comment (Can-Tina) [reply prisma/schema.prisma:25]: Definitely something to consider!

File: prisma/schema.prisma (hunk 3 above)
Comment (Can-Tina) [reply prisma/schema.prisma]: Ah that was something the schema did on its own, I forgot to correct it. Will do now

File: prisma/schema.prisma (hunk 4 above)
Comment (Can-Tina) [reply prisma/schema.prisma]: Same as last response

File: prisma/schema.prisma (hunk 1 above)
Comment (vherus) [reply prisma/schema.prisma:25]: I completely agree with Nathan, here. Well done, Nathan.

Usually you won't store counts of things in a db table directly - it's almost always a calculation done on the fly. You'd display the number of ratings a thing has by counting how many `Rating` records exist for the thing in question.

You can move forward with this path you've chosen if you wish, there will be learnings to be had by doing it... remember to update this ratingCount every time a rating is added or removed :)

File: prisma/schema.prisma [hunk 5]
```diff
@@ -12,9 +12,93 @@ datasource db {
...
+  updatedAt      DateTime         @updatedAt
+  userTeamMember UserTeamMember[]
+  rating         Rating[]
+}
+
+model User {
+  id        Int      @id @default(autoincrement())
+  username  String   @unique
+  password  String
+  email     String
```
Comment (vherus) [inline prisma/schema.prisma]: Email should probably be unique

**Review by vherus — APPROVED** [review]
Definitely think about Nathan's comment about ratingCount, but you're free to move forward with this!

**Pull request merged**